### DISCUSSION TITLE
Is there a way to add "preload" to the CSS generated by dynamic() ?

### DISCUSSION THREAD

**Discussion by JorgeSivil** [vercel/next.js#15138] [Help]
PageSpeed Insights is giving me advice on reducing times by using "preload" in the CSS generated by "dynamic()" components.

The problem is that I trigger the initialization of this "lazy" component on page load for Mobile but in Desktop for the same page, it is not always shown (depends on some parameters).
So google detects this component as something "I use always" so suggests to use "preload".

Is there a way I can add the "preload" to the dynamic css?

**Comment by mauriciocarnieletto** [comment]
Would love to know
